File: app/src/main/java/com/example/tics/ui/classes/Tic.java
package com.example.tics.ui.classes;

public class Tic {
    private int TicCount;
    private String TicDate;
    private String TicTime;
    private byte[] Picture;

    public Tic(int TicCount, String TicDate, String TicTime, byte[] Picture) {
        this.TicCount = TicCount;
        this.TicDate = TicDate;
        this.TicTime = TicTime;
        this.Picture = Picture;
    }

    public int getTicCount() {
        return TicCount;
    }

    public String getTicDate() {
        return TicDate;
    }

    public String getTicTime() {
        return TicTime;
    }

    public byte[] getPicture() {
        return Picture;
    }
}

File: php/tic.php
<?php
require "conn.php";
$studentID = $_GET['StudentID'];

$sql = "SELECT TicCount, TicDate, TicTime, Picture FROM Tic WHERE StudentID = '$studentID' ORDER BY TicDate DESC, TicTime DESC";

$result = $conn->query($sql);
$tics = array();
if ($result->num_rows > 0) {
    while ($row = $result->fetch_assoc()) {
        $tic = array(
            'TicCount' => $row['TicCount'],
            'TicDate' => $row['TicDate'],
            'TicTime' => $row['TicTime'],
            'Picture' => base64_encode($row['Picture'])
        );
        array_push($tics, $tic);
    }
}
echo json_encode($tics);

$conn->close();
?>